assets/js/modules/adsense/components/dashboard/AdBlockingRecoveryWidget.test.js

```diff
@@ -74,6 +71,7 @@ describe( 'AdSenseConnectCTA', () => {
 				slug: 'adsense',
 			},
 		] );
+		registry.dispatch( CORE_USER ).setReferenceDate( '2023-06-15' );
```

Comment:
Then update this to use `referenceDate`.

```suggestion
		registry.dispatch( CORE_USER ).setReferenceDate( referenceDate );
```